### PR TITLE
Cross

### PR DESCRIPTION
**Pull request by AlbertoElias**

### CODE REVIEW

File: src/js/overlay.js [hunk 1]
```diff
@@ -156,9 +156,11 @@ Overlay.prototype.render = function() {
 			heading.classList.add('o-overlay__heading--shaded');
 		}
 
-		var button = document.createElement('button');
+		var button = document.createElement('a');
 		button.className = 'o-overlay__close';
-		var buttonIcon = document.createElement('div');
+		button.setAttribute('role', 'button');
+		button.setAttribute('href', '#void');
+		var buttonIcon = document.createElement('span');
```
Comment (kaelig) [inline src/js/overlay.js]: For accessibility, please add (sorry I forgot them):

```js
button.setAttribute('aria-label', 'Close');
button.setAttribute('title', 'Close');
```

File: src/scss/_cross.scss [hunk 2]
```diff
@@ -0,0 +1,114 @@
+/////
+/// @group o-overlay
+/// @link http://registry.origami.ft.com/components/o-overlay
+////
+
+/// Helper mixin to target IE8
+///
+/// @access private
+@mixin _targetIE8 {
```
Comment (kaelig) [inline src/scss/_cross.scss]: The mixin's name should be `_oOverlayTargetIE8`

**Comment by kaelig** [comment]
Looks good to me :+1:

**Comment by AlbertoElias** [comment]
@kaelig The fixes are in :)

**Comment by kaelig** [comment]
:ship: it!

**Pull request merged**